src/components/subscriptions/edit/EditSubAddons.js

```diff
@@ -49,158 +64,230 @@ function EditSubAddonsDialog(props) {
         open,
         parentId,
         selectedSubscriptionAddons,
+        subscriptionId,
     } = props;
     const { t } = useTranslation("subscriptions");
+    const queryClient = useQueryClient();
+    const [subAddonUpdateError, setSubAddonUpdateError] = useState(null);
+
+    const { mutate: updateSubAddon } = useMutation(
+        (submission) => putSubAddon(subscriptionId, submission),
+        {
+            onSuccess: () => {
+                queryClient.invalidateQueries(SUBSCRIPTION_ADDONS_QUERY_KEY);
+                announce({ text: t("subAddonsUpdated") });
+                onClose();
+            },
+            onError: { setSubAddonUpdateError },
```

Comment:
This looks like a typo, since `onError` should be a function rather than an object:
```suggestion
            onError: setSubAddonUpdateError,
```

Reply:
Good catch!